pkg/cidata/cidata.TEMPLATE.d/boot.sh

```diff
@@ -77,6 +77,64 @@ if [ -d "${LIMA_CIDATA_MNT}"/provision.data ]; then
 	done
 fi
 
+if [ -d "${LIMA_CIDATA_MNT}"/provision.yq ]; then
+	yq="${LIMA_CIDATA_GUEST_INSTALL_PREFIX}/bin/lima-guestagent yq"
```

Comment:
Guestagent is only installed when `LIMA_CIDATA_PLAIN` is not set. I assume the provisioning mode should work for plain instance too, so need to execute it from the cidata mount point.

Also need to make sure the guestagent is included in the CIDATA iso for plain instances if there is at least one `yq` provisioning entry.

```suggestion
	yq="${LIMA_CIDATA_MNT}/lima-guestagent yq"
```